src/scripts/persist-indicators/config.json

```diff
@@ -2,6 +2,7 @@
     "auth": "YXF1aW50YW5hOkFxdWludGFuYTIwMTY=",
     "protocol": "http",
     "url": "localhost:8080/api",
+    "conf.apiversion": "24",
```

Comment:
@josemp10 I would change the name to apiVersion. It is already a config file and we are using camel case format for the whole file. Can you please change it?